src/controllers/trip.controller.js

```diff
@@ -1,8 +1,8 @@
 import TripService from '../services/trip.service';
 import ResponseService from '../services/response.service';
-import JwtService from '../services/jwt.service';
 import RequestService from '../services/request.service';
 import UserService from '../services/user.service';
+import paginationHelper from '../helpers';
```

Comment:
The paginationHelper function shouldn't be the default export but a named export

Reply:
Here because the file has only one function when I use a named export eslint throws an error that it prefers the default export